packages/react/src/runtimes/utils/MessageRepository.tsx

```
@@ -18,13 +21,30 @@ export type ExportedMessageRepositoryItem = {
+
+export const ExportedMessageRepository = {
+  fromArray: (
+    messages: readonly ThreadMessageLike[],
+  ): ExportedMessageRepository => {
+    const conv = messages.map((m) =>
+      fromThreadMessageLike(m, generateId(), getAutoStatus(false, false)),
+    );
+
+    return {
+      messages: conv.map((m, idx) => ({
+        parentId: idx > 0 ? conv[idx - 1]!.id : null,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)